app/src/main/res/values/strings.xml

```diff
@@ -142,8 +142,8 @@
         The first parameter is the search term that the user used. (for example: your search for "cat")-->
     <string name="recent_tabs_search_term">Your search for \"%1$s\"</string>
     <!-- Text for the number of tabs in a group in the 'Jump back in' section of the new tab
-        The first parameter is the count for number of sites in the group.  This number will always be more than one. -->
-    <string name="recent_tabs_search_term_count">Sites: %1$s</string>
+        %d is a placeholder for the number of sites in the group. This number will always be more than one. -->
+    <string name="recent_tabs_search_term_count_2">%d sites</string>
```

Comment:
@Delphine @Mugurell I am wondering if this will break localization for Beta since we can't just remove a string like this iirc

Reply:
Right, forgot about https://github.com/mozilla-mobile/fenix/issues/20871.
Will add add the previous string back, thank you!

Reply:
Opened https://github.com/mozilla-mobile/fenix/pull/22572 to add back the previous string.